specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-10-01/examples/ExternalSubscriptionsDimensions.json

```diff
@@ -0,0 +1,43 @@
+{
+  "parameters": {
+    "api-version": "2019-10-01",
+    "externalSubscriptionId": "100",
+    "scope": "providers/Microsoft.CostManagement/externalSubscriptions/123"
+  },
+  "responses": {
+	"200": {
+		"body": {
+			"value": [
+				{
+				"id": "providers/Microsoft.CostManagement/externalSubscriptions/123/dimensions_ResourceType_2019-12-01_2019-12-31",
+				"name": "dimensions_ResourceType_2019-12-01_2019-12-31",
+				"type": "microsoft.consumption/dimensions",
+				"properties": {
+					"data": [],
+					"total": 0,
+					"category": "ResourceType",
+					"description": "Resource type",
+					"usageStart": "2019-12-01T00:00:00Z",
+					"usageEnd": "2019-12-31T00:00:00Z",
+					"filterEnabled": true,
+					"groupingEnabled": true
+					}
+				}, 
+				{
+					"id": "providers/Microsoft.CostManagement/externalSubscriptions/123/dimensions_ResourceId_2019-12-01_2019-12-31",
+					"name": "dimensions_ResourceId_2019-12-01_2019-12-31",
+					"type": "microsoft.consumption/dimensions",
+					"properties": {
+						"data": [],
+						"total": 0,
+						"category": "ResourceId",
+						"description": "Resource ID",
+						"usageStart": "2019-12-01T00:00:00Z",
+						"usageEnd": "2019-12-31T00:00:00Z",
+						"filterEnabled": true,
+						"groupingEnabled": true
+					}
+				}]
+			}
+		}
+	}
```

Comment:
Missing one close curly brace here, if you count carefully.

Reply:
Thank you for pointing this out, I've added the missing curly bracket